app/Http/Controllers/Admin/CallchampionsController.php

```diff
@@ -84,42 +83,63 @@ public function update_call($due_id_encrypted){
 		$helper_obj = new Helpers;
 		$dueid = $helper_obj->decode($due_id_encrypted);
 		$duelist = DueList::where('due_id',$dueid)->get();
-		$b_id = $duelist[0]['fk_b_id'];
 		$action_items = Input::get('action_item');
 		$call_stats = Input::get('callstats');
 		$general_note = Input::get('general_note');
+		$duedate_stats = Input::get('duedatestat');
 		$cc = new CallChampion;
 
-		$data['callchampion'] = User::where('user_id',$this->user_id)->get();
-		$b_obj = new Beneficiary();
-		$data['beneficiary'] = $b_obj->get_beneficiary_details($b_id);
-
-		$email = 'mothercare@sevasetu.org';
-		if($call_stats == 'Incorrect number')
-		{
-			$data['action'] = " has marked an Incorrect number status.";
-			Mail::send('emails.admin_notification',$data, 
-						function($message) use($email){
-							$message
-							->to($email)
-							->subject('Seva Setu: Admin Notifications');
-						}
-					  );
-		}
-		if($action_items != trans('routes.textareadefaulttext') && strlen($action_items)>0)
+		if(sizeof($duelist))
```

Comment:
@yogikaushik  sizeof is not the best way to check whether the due list has an element.
the right way would be to first check if it's an array and then whether its empty, two php functions which can be ANDed.

else you could show me any regaular usage of duelist. 
be careful with php - things which seemingly work will end up crashing when you least expect it. please use tried and tested constructs only.